src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -1041,46 +1059,49 @@ class LicenceService(
     )
   }
 
+  @RequiresCom("Do we timeout a time served case?, Do we timeout a licence if it has no COM allocated?")
   @Transactional
   fun timeout(licence: EntityLicence, reason: String? = null) {
     check(licence is SupportsHardStop) { "Can only timeout licence kinds that support hard stop: ${licence.id}" }
+    if (licence is HasCom) {
```

Comment:
you can remove this as this is covered by the above check

Reply:
Removing this causes an issue when we call `sendEditedLicenceTimedOutEmail` to send the email as it can't find the COM on the entity Licence. Fixed by moving the check to that specific location only.